plugins/sql_functions.cc

```diff
@@ -366,7 +366,7 @@ SqlExpressionFunction(MldbServer * owner,
 
         // 4.  Our required input is known by the binding context, as it records
         //     what was read.
-        info.input = innerScope->inputInfo;
```

Comment:
This should use the ExpressionValueInfo::toRow() function, not a dynamic pointer cast.  (blocker)